### ISSUE TITLE
Replaced Forms are still there and are not getting deleted

### ISSUE THREAD

**Issue by MartinH0** [mikehaertl/php-pdftk#209]
I created a PDF Template with fillable forms and filled them with PHP.
I also run "flatten" but it just changed the forms from fillable to non fillable.
This is obviously ok, but each placeholderform will stay there forever and just gets "overwirtten" with a new text which will not write INTO the placeholder but above it.

I will attatch a Screenshiot from a PDF which I later then inspected with Adobe Acrobat Pro:
![PDF-TK Bug](https://user-images.githubusercontent.com/17692860/95970271-0a4cc100-0e10-11eb-9906-9f6b9d125a9b.PNG)

Here we can clearly see the date wriotten where the placeholder was, but the placeholder ist still there, just flatten out.

So basically I'm new to this and wanted to ask if I'm doing something wrong. And if yes, how to achove to not just overwrite the placeholder with new textareas, but to fill them?

Best regards

**Comment by mikehaertl** [comment]
Have you tried `dropXfa()`? See the comment here: https://github.com/mikehaertl/php-pdftk/blob/793c08b891dc9f197058135e6d5d7439b985a2f7/src/Pdf.php#L483-L488

**Comment by mikehaertl** [comment]
Apart from that I can not really provide more help. Form filling works, if the PDF is created properly for pdftk. I am not the author of pdftk, though, so if it doesn't work I have no idea.

**Comment by MartinH0** [comment]
Thank you very much!
I now added this function too, in two ways but it did not remove the empty dupicated un-needed forms.

Is there a official repo for PHP from PDFTK?
If so please redirect me to them so I can file a bugreport there.

**Comment by MartinH0** [comment]
Will this repo also work with "pdftk-java"?
As the normal PDFTK is not available anymore for Debian 10? It is marked as deprecated and will be replaced with "pdftk-java".

Would be awesome to get any infos about this

**Comment by mikehaertl** [comment]
> Is there a official repo for PHP from PDFTK?

This has nothing to do with PHP. `pdftk` is a standalone comande line tool. This library is just a frontend for that. We can't do anything that pdftk can't do (we only have some utility class for form data).

> Will this repo also work with "pdftk-java"?

I've never tried it. Also see #208.

**Comment by MartinH0** [comment]
I will try it and report back if it is compatible with pdftk-java.
Would be good, so this project can life long.
